File: backend/src/main/java/com/drip/admin/modules/system/vo/AuthMeVo.java
package com.drip.admin.modules.system.vo;

import java.util.List;

public record AuthMeVo(
    Long userId,
    String username,
    String realName,
    String phone,
    String email,
    String avatar,
    Long deptId,
    List<String> roles,
    List<String> permissions,
    List<MenuTreeVo> menus
) {
}

File: backend-go/internal/modules/system/vo/auth_me_vo.go
package vo

import (
	"drip-admin/backend-go/internal/common"
)

type AuthMeVo struct {
	ID          common.Int64String  `json:"id"`
	Username    string              `json:"username"`
	RealName    string              `json:"realName"`
	Phone       *string             `json:"phone"`
	Email       *string             `json:"email"`
	Avatar      *string             `json:"avatar"`
	DeptID      *common.Int64String `json:"deptId"`
	Roles       []string            `json:"roles"`
	Permissions []string            `json:"permissions"`
	Menus       []MenuTreeVo        `json:"menus"`
}
